newIDE/app/src/GameDashboard/PublicGameProperties.js

```diff
@@ -57,55 +66,91 @@ function PublicGameProperties({
   playWithMobile,
   setOrientation,
   orientation,
+  setDiscoverable,
+  discoverable,
+  displayThumbnail,
+  thumbnailUrl,
 }: Props) {
   const [categoryInput, setCategoryInput] = React.useState('');
 
   return (
     <I18n>
       {({ i18n }) => (
         <ColumnStackLayout noMargin>
-          <SemiControlledTextField
-            floatingLabelText={<Trans>Game name</Trans>}
-            fullWidth
-            type="text"
-            value={name}
-            onChange={setName}
-            autoFocus
-          />
-          {setCategories && (
-            <SemiControlledMultiAutoComplete
-              hintText={t`Select a genre`}
-              floatingLabelText={<Trans>Genres</Trans>}
-              helperText={
+          <Line noMargin>
```

Comment:
Take a look at what happens on a small screen. You might want to use a ResponsiveLineStackLayout to go to a 1 column stack on small screens automatically (the message "To update your thumbnail" might need to be moved to a small component and have its place changed according to the window size though..).

Reply:
As I think mobile is not the first use, I'll leave the message where it is for the moment. If this changes, this will be a nice problem to fix :)